x-pack/plugins/alerting/server/saved_objects/index.ts

```diff
@@ -185,12 +186,9 @@ export function setupSavedObjects(
     mappings: {
       dynamic: false,
       properties: {
-        // shape is defined in x-pack/plugins/alerting/server/data/ad_hoc_run/types/ad_hoc_run.ts
-        // TODO to allow invalidate api key task to query for backfill jobs still
-        // using the API key
-        // apiKeyId: {
-        //   type: 'keyword'
-        // },
+        apiKeyId: {
```

Comment:
This PR is merging into a feature branch. I have the model versions defined in [another branch](https://github.com/elastic/kibana/pull/179975) that is merging into the feature branch and the final PR (of the feature branch into `main`  should contain all the V1 mappings & schemas for this new saved object type.